src/os/bluestore/fastbmap_allocator_impl.cc

```diff
@@ -0,0 +1,397 @@
+#include "fastbmap_allocator_impl.h"
```

Comment:
pelase include the modelines at the top of the file.. it avoids some of the whitespace inconsistencies below (tab vs 8 spaces)